### PR TITLE
Add Webots Privacy Policy

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Address #1652:
add privacy policy for transparency and to be compliant with GDPR.

I also added a link to the privacy policy from the "About" window:
about

### CODE REVIEW

**Comment by omichel** [comment]
I realize that this about box looks pretty bad. In particular, it misses the version information which anybody expect to see in a about box. So, above the copyright notice, it would be nice to see with a slightly bigger font "Webots R2020b revision 1". What do you think? Also I would remove "Our" in "Our Website".

**Comment by stefaniapedrazzi** [comment]
Here is a new proposal for the About box.
I took inspiration from the one of Blender.

about2

**Comment by omichel** [comment]
It looks very good to me. Does the date correspond to the compilation date?

**Comment by stefaniapedrazzi** [comment]
> It looks very good to me. Does the date correspond to the compilation date?

Yes, it should correspond to the compilation date.
It is computed based on the `WbApplicationInfo::releaseDate` (that uses the ``__DATE__`` macro).

File: docs/guide/privacy-policy.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+## Privacy Policy
+
+Cyberbotics respects your privacy and it is committed to protecting privacy and personal data.
+This Privacy Policy applies to the personal data of "Webots" software that may be collected through [https://cyberbotics.com](https://cyberbotics.com).
```
Comment (omichel) [inline docs/guide/privacy-policy.md:4]:
```suggestion
This Privacy Policy applies to the personal data of the "Webots" software that may be collected through [https://cyberbotics.com](https://cyberbotics.com).
```

File: docs/guide/privacy-policy.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+### Personal Data Collected
+
+Personal data may be collected by the Webots telemetry system if activated from the Webots preferences or at first launch.
+The telemetry system will collect information about the system configuration, the IP address, the Webots settings and the Webots world name.
+
+### Use of Personal Data
+
+All the collected personal information are only used internally by Cyberbotics to improve the Webots software.
+After being anonymized, the collected personal information are published for on the telemetry page on [https://cyberbotics.com](https://cyberbotics.com).
```
Comment (omichel) [inline docs/guide/privacy-policy.md:15]:
```suggestion
After being anonymized, the collected personal information are published on the telemetry page at [https://cyberbotics.com/telemetry](https://cyberbotics.com/telemetry).
The purpose of this publication is openness and transparency.
It allows anyone to consider this data to better contribute to the Webots software.
```

File: docs/guide/privacy-policy.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+### Retention of Personal Data
+
+Cyberbotics S.à.r.l retains your personal information only for as long as necessary to comply with our legal obligations, resolve disputes, and enforce our agreements; and for so long as is necessary for the purposes for which we collected such personal information.
+
+### Rights of the Users
+
+You have the right, at any time, to request access, rectification and erasure of your personal information.
+In addition, you have the right to withdraw your consent, without this affecting the lawfulness of processing based on consent before its withdrawal.
+To exercise your rights under these provisions, please contact us at the “Communications” details below.
```
Comment (omichel) [inline docs/guide/privacy-policy.md:37]:
```suggestion
To exercise your rights under these provisions, please contact us (see “Communications” section below).
```

File: docs/guide/privacy-policy.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+Cyberbotics reserves the right to make changes to this privacy policy at any time by given notice to its Users on this page.
+It is strongly recommended to check this page often, referring to the date of the last modification listed at the bottom.
+If a User objects to any of the changes to the Policy, the User may disable the telemetry system and can request Cyberbotics to erase the personal data.
+Unless stated otherwise, the then-current privacy policy applies to all personal data Cyberbotics has about Users.
+
+### Communications
+
+If you have questions regarding this Privacy Policy, please contact us at [info@cyberbotics.com](mailto:info@cyberbotics.com).
+<br/>
+Latest update: September 10, 2020.
```
Comment (omichel) [inline docs/guide/privacy-policy.md:54]:
```suggestion
Latest update: September 14th, 2020.
```

File: src/webots/gui/WbAboutBox.cpp [hunk 5]
```diff
@@ -24,38 +26,49 @@ WbAboutBox::WbAboutBox(QWidget *parent) : QDialog(parent) {
...
-  licenseInfo->setWordWrap(true);
-  licenseInfo->setText(QString("<br> © Cyberbotics 1998-%1. Licensed under the <a style='color: #5DADE2;' "
-                               "href='https://www.apache.org/licenses/LICENSE-2.0'>Apache License, Version 2.0</a>.")
-                         .arg(year));
-  layout->addWidget(licenseInfo, 0, 1, 1, 1, Qt::AlignCenter);
+  QLabel *versionInfo = new QLabel();
+  connect(versionInfo, &QLabel::linkActivated, &WbDesktopServices::openUrl);
+  QDateTime releaseTimestamp;
+  releaseTimestamp.setTime_t(WbApplicationInfo::releaseDate());
+  versionInfo->setText(QString("<p style='font-size: large;'><b>%1</b></p>"
```
Comment (omichel) [inline src/webots/gui/WbAboutBox.cpp:42]: It looks better to me if Webots is printed before the version number...
```suggestion
  versionInfo->setText(QString("<p style='font-size: large;'><b>Webots %1</b></p>"
```

**Review by omichel — CHANGES_REQUESTED** [review]
Thank you.
I have a few suggestions...

**Review by omichel — APPROVED** [review]
Thank you.

**Comment by omichel** [comment]
It seems that for some users, the display of the date seems wrong (probably related with the locale):
webots R2020b rev2

**Comment by DavidMansolino** [comment]
> probably related with the locale

Most probably due to the use of `Qt::SystemLocaleShortDate`.

**Comment by stefaniapedrazzi** [comment]
i will change so that it will be independent from the locale settings.

**Pull request merged**